### PR TITLE
[BD-05] Bug fix: TinyMCE version conflicts between ORA and edx-platform

### PR DESCRIPTION
**Pull request by giovannicimolin**
**TL;DR -** This PR fixes TinyMCE prompt editor when the WYSIWYG editor is enabled in the responses field.

JIRA: [JIRA-XXXX](https://openedx.atlassian.net/browse/JIRA-XXXX)

**What changed?**

- [ More in depth breakdown of changes ]
- [ Peripheral things that got changed ]
- [ etc... ]

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @giovannicimolin! I've created [BLENDED-777](https://openedx.atlassian.net/browse/BLENDED-777) to keep track of it in Jira.   More details are on the [BD-05](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529970851/BD-05+ORA+v3+Usability+Moderation) project page.


When this pull request is ready, tag your edX technical lead.
